src/components/footer/footer.js

```diff
@@ -1,62 +1,38 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import Logo from "../shared/Logo/logo";
+import { NavLink } from "react-router-dom";
+import { IoMdSend } from "react-icons/io";
+import LogoWhite from "../../assets/icons/LogoWhite.png";
 import "./footer.scss";
-import constants from "../../constants/Footer";
+import { FOOTER_LABELS } from "../../constants/footerConstants";
+import { MapperSocialLink, MapperLink } from "./footer-mapper";
 
-const { SUBSCRIBE_US, SUBSCRIBE, CONTACT_US, PHONE_NUMBER } = constants;
-
-const links = [
-  { id: 0, name: "Home", path: "/home" },
-  { id: 1, name: "Search Event", path: "/events" },
-  { id: 2, name: "Create Event", path: "" },
-  { id: 3, name: "Terms", path: "/terms" },
-  { id: 4, name: "About", path: "/about" },
-];
-
-const socialLinks = [
-  { id: 0, icon: "fab fa-facebook-f", link: "https://uk-ua.facebook.com" },
-  { id: 1, icon: "fab fa-instagram", link: "https://www.instagram.com" },
-  { id: 2, icon: "fab fa-youtube", link: "https://www.instagram.com" },
-];
+const { SUBSCRIBE_US, CONTACT_US, PHONE_NUMBER, PASTE_YOUR_EMAIL, HOME } =
+  FOOTER_LABELS;
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_container">
+        <NavLink to={HOME}>
+          <img src={LogoWhite} alt="Logo" className="footer__logo" />
+        </NavLink>
+        <div className="footer__links">{MapperLink("footer__link_item")}</div>
```

Comment:
Is there any another usage of MapperLink, that requires us to manually pass className?
If we use it only one time, it doesn't require to pass it inside. Just specify it directly into the specific tag

Reply:
Done